data-raw/organizations/MIA/prepare-MIA.R

```diff
@@ -0,0 +1,45 @@
+# MIA Preparation Script
+
+# This is a template for importing, cleaning, and exporting data
+# ready for the many packages universe.
+
+# Stage one: Collecting data
+MIA <- haven::read_dta("data-raw/organizations/MIA/DP_May 2021.dta")
+
+# Stage two: Correcting data
+# In this stage you will want to correct the variable names and
+# formats of the 'MIA' object until the object created
+# below (in stage three) passes all the tests.
+MIA <- as_tibble(MIA) %>%
+  dplyr::rename(igoID = acronym,
+                COWcode = ionumber) %>%
```

Comment:
Shouldn't COWcode be cowID or similar? It would make identifying overlaps between the different datasets in the database easier...

Reply:
I've changed it to `cowigoNR` in both `COW` and `MIA`, using NR instead of ID to distinguish it from the three-letter abbreviations used for IGOs and states. In `manystates`, `cowID` refers to the three-letter abbreviation of states' names.